### PR TITLE
Temp8 - Fixing spelling / typos etc in Umbraco.Web

### PR DESCRIPTION
**Pull request by readingdancer**
### Prerequisites

Issue  #4147 

### Description
As described in the issue, this pull request fixes a lot of spelling mistakes, typos and ambiguous abbreviations throughout the Umbraco.Web project. I have tried to ensure that only code comments and text strings that would not be breaking changes have been updated.

I have also updated all the "TODO" comments so they are written in a consistent fashion and ensured that they all show up correctly in the Visual Studio Task list.

I did a rebase before committing and as you can see I screwed it up slightly, hence the two additional commits to remove two files that I had accidentally re-added.  ( they have been removed in one of the recently merged commits )

### CODE REVIEW

**Comment by readingdancer** [comment]
Can @nul800sebastiaan or the #PRTeam please remind me, for pull requests like this that have not been commented on ( or any that have not been merged ) do you expect the person who submitted the pull request to keep them rebased, or should we leave them for you to do? ( assuming you decide to merge in the PR )

**Comment by nul800sebastiaan** [comment]
Thanks @readingdancer - for all PRs it's super helpful if you can keep them up to date if they happen to get out of sync. Unfortunately we haven't had time to merge this one yet, we'll get to it!

**Comment by readingdancer** [comment]
@nul800sebastiaan I've just rebased this one, not sure why, but I had to force push the branch, but it all looks ok :)

**Comment by readingdancer** [comment]
This was still showing as having conflicts with a handful of files, they were all related to comments so I've just used Github to resolve them all which seemed quite easy. I do hope you can merge these two Spelling PR's in quickly as I hate having to rebase and fix merge conflicts ;-)

File: src/Umbraco.Web/PublishedElementExtensions.cs [hunk 1]
```diff
@@ -146,57 +146,6 @@ public static T Value<T>(this IPublishedElement content, string alias, string cu
 
         #endregion
```
Comment (readingdancer) [inline src/Umbraco.Web/PublishedElementExtensions.cs]: @nul800sebastiaan I've just updated the three conflicting files to keep the branch up to date so you can hopefully merge it in, however I just thought it was worth pointing out this section where it seems this code was missing in the Temp8 branch. So I have also removed it here, do confirm this is correct :)

**Comment by nul800sebastiaan** [comment]
🔤 ✅

**Pull request merged**